src/components/navigation/footer/Footer.tsx

```diff
@@ -29,13 +29,13 @@ interface Footer extends React.FC<FooterProps> {
 }
 
 const Footer: Footer = ({ className, children, visuallyHiddenText = 'Support links', ...rest }) => (
-  <footer {...rest}>
-    <div className={classNames('nhsuk-footer', className)}>
+  <footer role="contentinfo" {...rest}>
+    <div className={classNames('nhsuk-footer-container', className)}>
       <Container>
         {visuallyHiddenText ? (
           <h2 className="nhsuk-u-visually-hidden">{visuallyHiddenText}</h2>
         ) : null}
-        {children}
+        <div className="nhsuk-footer">{children}</div>
```

Comment:
slight issue in styling with single column

Reply:
Resolved, have had to do some gymnastics with cloning elements and but now it only adds the specific class we need for this if there is only one column